web-ng/src/app/components/map/map.component.ts

```diff
@@ -294,6 +320,30 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private addPolygonToMap(feature: PolygonFeature) {
+    const vertices: google.maps.LatLng[] = [];
+    feature.polygonVertices.map(vertex => {
+      vertices.push(new google.maps.LatLng(vertex.latitude, vertex.longitude));
+    });
+    // Construct the polygon.
+    const polygon = new google.maps.Polygon({
+      paths: vertices,
+      strokeColor: '#ff9131',
```

Comment:
The color should refer to the color of the layer right?

Reply:
replaced with project.layers.get(feature.layerId)?.color